locales/en/yourloops.json

```diff
@@ -107,13 +107,15 @@
   "caregiver-and-family": "Caregiver and family",
   "caregivers-title": "My Caregivers",
   "certified-professional-account": "Certified professional",
+  "change-columns-settings": "Change columns settings",
   "change-password-info": "By clicking this button, you will receive an e-mail allowing you to change your password.",
   "chat-footer-private": "Private",
   "chat-footer-reply": "Reply",
   "chat-footer-start-writing": "Start typing",
   "coefficient-of-variation": "CV ({{cbgLabel}})",
   "coefficient-of-variation-tooltip": "**CV (Coefficient of Variation):** The ratio of the standard deviation to the mean glucose. For any period greater than 1 day, we calculate the mean of daily CV.",
   "colon": "{{label}}:",
+  "columns-settings-cannot-changed-pending-tab": "Columns settings cannot be changed while on the pending tab",
```

Comment:
<!-- Nitpicks are trivial preference-based requests. These should be non-blocking by nature.  -->
**🤓 nitpick:** ‏We could keep the same structure as the sentence for the filters

```json
"columns-settings-cannot-changed-pending-tab": "Columns settings cannot be changed on Pending tab",
```